pkg/operator/controller.go

```diff
@@ -676,6 +677,11 @@ func (c *Controller) syncProfile(tuned *tunedv1.Tuned, nodeName string) error {
 		return fmt.Errorf("failed to get ProviderName: %v", err)
 	}
 
+	// Won't be set in case of operator restart
```

Comment:
Can we improve the terse comment a bit?  I know it is obvious what won't be set, but perhaps we want to be more explicit?